WETH input broken in swapper

### Overview

WETH of all flavors is broken as an input asset on the swapper

### References and additional details

1. select WETH on ETH (or another chain)
2. attempt to get a rate 
3. see we show the following console err
```
2apiState.ts:870 Error: missing sellAssetUsdRate
    at Object.queryFn (swappersApi.ts:53:38)
    at apiState.ts:804:65
    at apiState.ts:23:23
    at Object.next (apiState.ts:4:53)
    at apiState.ts:85:65
    at new Promise (<anonymous>)
    at S (apiState.ts:67:12)
    at f (apiState.ts:768:54)
    at index.ts:1237:61
    at index.ts:38:23
```

### Acceptance Criteria

- WETH works in the swapper as an input asset 

### Need By Date

_No response_

### Screenshots/Mockups

_No response_

### Estimated effort

_No response_

--- a/src/state/apis/swappers/swappersApi.ts
+++ b/src/state/apis/swappers/swappersApi.ts
@@ -47,15 +47,17 @@ export const swappersApi = createApi({
         } = getTradeQuoteInput
         const isCrossAccountTrade = sendAddress !== receiveAddress
         const featureFlags: FeatureFlags = selectFeatureFlags(state)
-        const sellAssetUsdRate = selectUsdRateByAssetId(state, getTradeQuoteInput.sellAsset.assetId)
         const enabledSwappers = getEnabledSwappers(featureFlags, isCrossAccountTrade)
 
-        if (!sellAssetUsdRate) throw Error('missing sellAssetUsdRate')
-
-        // Await market data fetching thunk, to ensure we can display some USD rate and don't bail in getDependencies above
+        // hydrate crypto market data for buy and sell assets
         await dispatch(
           marketApi.endpoints.findByAssetIds.initiate([sellAsset.assetId, buyAsset.assetId]),
         )
+
+        const sellAssetUsdRate = selectUsdRateByAssetId(state, getTradeQuoteInput.sellAsset.assetId)
+
+        // this should never be needed but here for paranoia
+        if (!sellAssetUsdRate) throw Error('missing sellAssetUsdRate')
 
         // We use the sell amount so we don't have to make 2 network requests, as the receive amount requires a quote
         const isDonationAmountBelowMinimum = getIsDonationAmountBelowMinimum(

